core/src/main/kotlin/utils/JobMetrics.kt

```diff
@@ -113,4 +174,94 @@ class JobMetrics(private val application: Application) : MeterBinder {
 
     private fun countScannerJobs(status: JobStatus) =
         transaction { ScannerJobDao.count(ScannerJobsTable.status eq status).toDouble() }
+
+    private fun getAnalyzerJobDurations(timestamp: Instant): List<Duration> =
+        transaction {
+            AnalyzerJobsTable.selectAll().where {
```

Comment:
A tiny optimization would be to select only `AnalyzerJobsTable.startedAt` and `AnalyzerJobsTable.finishedAt`.